py3status/modules/weather_yahoo.py

```diff
@@ -8,11 +8,29 @@
 Find your city code using:
     http://answers.yahoo.com/question/index?qid=20091216132708AAf7o0g
 
+Find your woeid using:
+    http://woeid.rosselliot.co.nz/
+
 Configuration parameters:
     - cache_timeout : how often to check for new forecasts
     - city_code : city code to use
+    - woeid : use Yahoo woeid (extended location) instead of city_code
     - forecast_days : how many forecast days you want shown
     - request_timeout : check timeout
+    - units : Celsius (C) of Fahrenheit (F)
+    - today_format : possibple placeholders: {icon}, {temp}, {units}, {text}
+        example:
```

Comment:
I would suggest changing this example to °{units} and °C so that it is correct. Just a minor nitpick.